github/ci/prow/files/jobs/nmstate/kubernetes-nmstate-presubmits.yaml

```diff
@@ -93,3 +93,34 @@ presubmits:
               - "/bin/sh"
               - "-c"
               - "automation/check-patch.e2e-okd.sh"
+    - name: pull-kubernetes-nmstate-e2e-ocp
+      skip_branches:
+        - release-\d+\.\d+
+      annotations:
+        fork-per-release: "true"
+      always_run: true
```

Comment:
you want always to run ?

Reply:
Do you suggest we make it required, but only per request? Good point.

Reply:
hmm at least at the beginning i think, maybe even do it not always or at least optional ? 
lets keep in mind that every now and then a new provider exists, sometimes it take time to stabilize and we dont want it to block hard the pipe i think ?

Reply:
I agree with you. I made it optional and not always_run. Once we stabilize, we can switch it on.